src/Repository/FeedbackRepository.php

```diff
@@ -88,4 +89,12 @@ public function getUserFeedbackPaginated(
 
         return $query;
     }
+
+    public function canUserGiveFeedback(Activity $activity, User $userFrom, User $userTo): bool
```

Comment:
This method doesn't implement all the checks necessary, so it's name should be changed to `hasUserGivenFeedback()`